manifests/testing/kubevirt-config.yaml.in

```diff
@@ -7,3 +7,4 @@ metadata:
     kubevirt.io: ""
 data:
   feature-gates: "DataVolumes, CPUManager, LiveMigration, ExperimentalIgnitionSupport, Sidecar"
+  selinuxLauncherType: "virt_launcher.process"
```

Comment:
This is required for kubevirt to work with selinux enabled, right?

sp would it make sense to make this selinuxLauncherType the default internally rather than having to be explicitly set in the config?

Reply:
It is required if you plan on using network multiqueue or hugepages.
I implemented it this way to make it explicit that we're using a custom policy to work around issues with libvirt/k8s/container-selinux that will hopefully be resolved soon.
Also from an implementation perspective, we need a way to say "let the engine manage SELinux", which currently means container_t, but there's no reason that couldn't change... So I guess I would have to use a special value or something, because forcing container_t is different from letting the engine default to it. (https://github.com/kubernetes/kubernetes/issues/90759)